src/libcmis/oauth2-providers.cxx

```diff
@@ -33,6 +33,11 @@
 #include "oauth2-providers.hxx"
 #include "http-session.hxx"
 
+#define CHALLENGE_PAGE_ACTION "/signin"
```

Comment:
Well, what I had on mind was something like

#define STR_LIT_WITH_LEN(str) str, (sizeof(str) - 1)

, but this is good enough...